diorama.js

```diff
@@ -702,6 +708,38 @@ const createPlayerDiorama = ({
             localVector2.set(0, cameraOffset.y, 0)
               .applyQuaternion(targetQuaternion)
           );
+          // handle player diorama
+          const lerp = (a, b, x) => {
+              return a + (b - a) * x;
+          }
+          if(this.player){
+            const headPosition = localVector3.setFromMatrixPosition(this.player.avatar.modelBones.Head.matrixWorld);
+            if(this.player.hasAction('sit')){
+              sideCamera.position.y -= this.player.position.y - headPosition.y;
+            }
+            else{
+              const isNarutoRun = this.player.hasAction('narutoRun');
+              const isCrouch = this.player.hasAction('crouch');
+              
+              if(this.lastCrouch !== isCrouch || this.lastNR !== isNarutoRun){
+                this.lerpAlpha = 0.01;
+              }
+              else{
+                if(this.lerpAlpha < 1){
+                  this.lerpAlpha += 0.05;
+                }
+              }
+              sideCamera.position.x = lerp(sideCamera.position.x, sideCamera.position.x - (this.player.position.x - headPosition.x), this.lerpAlpha);

```

Comment:
This could be `.set`?

Reply:
https://github.com/webaverse/app/blob/32adf742456f9e16d8eab99ce488823362f5c834/diorama.js#L738

Reply:
change it to vector3.lerp 

https://github.com/webaverse/app/blob/32adf742456f9e16d8eab99ce488823362f5c834/diorama.js#L740